src/MockFactory.ts

```diff
@@ -27,6 +27,7 @@ const generateMockSchema = (schemaJson: any) => {
     if (
       typeName.startsWith('__') ||
       typeName === 'Query' ||
+      typeName === 'Mutation' ||
```

Comment:
What does this do?

Reply:
This code is responsible for adding query fields to the schema to be able to query each type. If the field that we're currently looking at is `Query` or `Mutation` then we don't do that